x-pack/legacy/plugins/task_manager/queries/mark_available_tasks_as_claimed.ts

```diff
@@ -14,7 +14,9 @@ import {
   RangeBoolClause,
 } from './query_clauses';
 
-export const RecuringTaskWithInterval: ExistsBoolClause = { exists: { field: 'task.interval' } };
+export const TaskWithRecurringSchedule: ExistsBoolClause = {
```

Comment:
nit: `TaskWithSchedule`

Reply:
No a nit at all, we changed that name and I missed it :)
Thanks!